.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v5
```

Comment:
There is a "sync" option we should use. This will allow the labeler to remove labels as well (when the file set changes)

Reply:
Not sure we actually need to enable this feature.
The labeler will remove labels even if some labels are not set by it (eg. a label comes from maintainer or other bot).

You can refer https://github.com/actions/labeler/issues/763

Reply:
Ah, that's too bad. Maybe we can enable that feature once the action has been fixed.